modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/engine/util/QueryChecker.java

```diff
@@ -251,6 +251,8 @@ public static Matcher<String> containsAnyScan(final String schema, final String
 
     private String exactPlan;
 
+    private boolean executeExplainPlan = true;
```

Comment:
It looks unclear for me. 
If `executeExplainPlan` is true, then will the query itself be excecuted, or just EXPLAIN for the query?
Maybe `skipExplan` will be more precise name.

Reply:
Yep. Going to fix this since it does not work anyway.